matminer/featurizers/dos.py

```diff
@@ -101,6 +101,97 @@ def implementors(self):
         return ['Maxwell Dylla', 'Alireza Faghaninia', 'Anubhav Jain']
 
 
+class DopingFermi(BaseFeaturizer):
+    """
+    Futurizes a pymatgen density of states (dos) object. structure is needed;
+    it can either be fetched automatically from dos.structure(e.g. CompleteDos)
+    or passed into featurize/featurize_dataframe. This featurizers return the
```

Comment:
Could you revise this doc string to make the 3rd sentence ("This featurizer returns...") the first? That way, the featurizer summary table Alex made (#200) will display more meaningful information.

Reply:
@albalu Preferably just put the first line of the docstring as:

Fermi level associated with input carrier concentration and temperature w.r.t.  reference energy